pylint_odoo/checkers/modules_odoo.py

```diff
@@ -398,7 +403,11 @@ def _is_module_name_in_whitelist(self, module_name):
     def _check_imported_packages(self, node, module_name):
         """Check if the import node is a external dependency to validate it"""
         if not module_name:
-            # skip local packages because is not a external dependency.
+            # skip local packages because is not a external dependency,
+            # except to check if the test folder is not being imported
```

Comment:
I'd just put the check before this comment

Reply:
In a new version, it's change f line.